app/src/main/java/com/github/epfl/meili/forum/NewPostActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.epfl.meili.forum
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import com.github.epfl.meili.R
+
+class NewPostActivity : AppCompatActivity() {
+
+    // Unique tag to tell where a log message came from
+    private val TAG = "NewPostActivity"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_new_post)
+    }
+
+    /** Called when the user taps the Create Post button */
+    fun createNewPost(view: View) {
+        // Get post information to store in database
+        val title = findViewById<EditText>(R.id.new_post_title).text.toString()
+        val text = findViewById<EditText>(R.id.new_post_text).text.toString()
```

Comment:
I think it's best practice to have these fields (newPostTitleView and newPostTextView) stored as private attributes of the activity, since they might be used multiple times.

Reply:
Ok

Reply:
I have to make the views be initialized in onCreate() though, so they will be latinit var. 

I am making the changes, it's just a heads up